docs/USAGE.md

```diff
@@ -12,19 +12,20 @@ how you may configure the settings.
 
 ## Emacs
 
- - Install [lsp-mode](https://github.com/emacs-lsp/lsp-mode)
- - Add the following to your `.emacs`:
+If you are using `use-package`, you can use something like this:
```

Comment:
> We should assume that even users who have not upgraded yet may land here

IIUC, MELPA's builds is not immutable. And it keeps replacing the older builds. So I don't the the older documentation is exactly right. I will push a fix where I will instead link the official documentation installation section.

> Would you mind linking to some docs about use-package?

I'm linking the official docs which has more details about use-package: https://emacs-lsp.github.io/lsp-mode/page/installation/#use-package

>  Can you clarify where does the config go exactly and/or link to some relevant docs?

It goes in the `init.el` file which is documented in the official doc. I can add it here, if you think it will help.

Reply:
Thanks - this is helpful.

I think you accidentally removed the example config too though?

> It goes in the init.el file which is documented in the official doc. I can add it here, if you think it will help.

The only mention of `init.el` I can find is https://emacs-lsp.github.io/lsp-mode/page/installation/#doom-emacs which I don't think is relevant in this particular context - so yes - I think it would be helpful to document that too, and/or link to some relevant docs upstream.

Reply:
I have updated it, can you give it a look ? Thanks!